vault/provider.go

```diff
@@ -466,6 +470,21 @@ var (
 			Resource:      ldapAuthBackendGroupResource(),
 			PathInventory: []string{"/auth/ldap/groups/{name}"},
 		},
+		"vault_nomad_secret_backend": {
+			Resource: nomadSecretAccessBackendResource(),
+			PathInventory: []string{
+				"/nomad",
+				"/nomad/config/access",
+			},
+		},
+		"vault_nomad_secret_lease": {
```

Comment:
I feel that the fields managed by this resource should be managed by `vault_nomad_secret_backend`, and this resource should be removed. From a user perspective, it makes more sense to have those values in a single resource as they pertain to just that resource.

Reply:
Merged